commonjs/tabs/tabs.js

```diff
@@ -116,7 +158,7 @@ Tabs.prototype = {
                 duration: this.fxDuration,
```

Comment:
I would like to discuss to remove the animations (and eventually replace them by CSS Transitions). Pros, Cons?